.github/workflows/pr.yaml

```
@@ -35,6 +35,32 @@ jobs:
         with:
           version: nightly
 
+      - name: Use Node.js 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install yarn
+        run: npm install --global yarn
+      
+      - name: Install dependencies
+        shell: bash
+        run: yarn --prefer-offline
+
+      - name: Check TypeScript files
+        if: ${{ contains(matrix.system.os, 'windows') == false }}
```

Executing the steps only for one of the OS should be fine!?
```suggestion
        if: ${{ contains(matrix.system.os, 'ubuntu') == true }}
```

---

Will be addressed in separate [Issue](https://github.com/LiskHQ/lisk-contracts/issues/210)